faq.md

```diff
@@ -380,6 +380,24 @@ framework, we've decomposed complex widgets to pieces that separately implement
 the visual, interaction, and motion design. You can remix these widgets however
 you like to make your own custom widgets that have full range of expression.
 
+### Why would I want to share layout code across iOS and Android?
+
+More and more, we see mobile app layouts and designs converging
+to be more brand-driven and unified across platforms. For example,
+the "bottom nav bar" pattern can now be naturally found across iOS and Android.
+The brand identity and customization of the app's aesthetic design is now
+becoming more important than strictly adhering to traditional platform aesthetics.
+
+Given the importantance on brand-driven design, plus the fact that
+modern apps often have many customized design elements (including branding,
+widgets, animations, and information architectures) sharing
+layout and UI code across platforms is a common practice.
+
+When designers choose to implement different app layouts for iOS and Android,
+Flutter makes that possible. Developers are free to check the mobile OS
+and render different layouts, though we find this to be not too common in
+practice.
```

Comment:
Maybe reverse the order here, starting with "you don't have to", and then moving on to "but you may want to because"?

Reply:
do you mean reverse the order of these last two sentences ? where is the "here" you are referring to? :)

Reply:
I meant the whole three paragraphs.

The question "why would i" implies "i don't want to". So the quickest way to diffuse the question is "you don't have to". But our value comes in part from the way that you can, so then following up with the rest of the answer here would provide the context to actually answer the question.

Right now it reads "Why would I do this dumb thing?" "I contradict you, it's not dumb, because you should do things different! bla bla bla bla bla. Oh also, you don't actually have to."

I'm suggesting having it read "Why would I do this dumb thing?" "You don't have to. You may want to because bla bla bla bla bla."